src/components/Proposals/ProposalDetails/VoteActions.js

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useCallback } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  GU,
+  IconCheck,
+  IconConnect,
+  IconCross,
+  Info,
+  noop,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { dateFormat, toMs } from '../../../utils/date-utils'
+
+import { useWallet } from '../../../providers/Wallet'
+
+export const VOTE_YEA = 'VOTE_YEA'
+export const VOTE_NAY = 'VOTE_NAY'
```

Comment:
Just wondering about the export here, are we intending to use these outside of this component?